src/shared-components/dropdown/index.tsx

```diff
@@ -50,15 +68,15 @@ class Dropdown extends React.Component {
     this.closeDropdown();
   };
 
-  onOptionClick = event => {
+  onOptionClick = (event: React.MouseEvent<HTMLLIElement>) => {
     const { onChange } = this.props;
-
-    if (event.target.hasAttribute('disabled')) {
+    const target = event.target as HTMLSelectElement;
```

Comment:
Since we're applying `onOptionClick` to each `option` we map over in the `DesktopDropdown`, we should be able to swap `event.target` with `event.currentTarget` (they should always be the same element) so that we can remove the casting.

<img width="303" alt="Screen Shot 2020-07-20 at 9 38 47 AM" src="https://user-images.githubusercontent.com/13544620/87950392-de1de000-ca6c-11ea-88df-cd87e7afb1b0.png">